app/assets/javascripts/models/beta/story.js

```diff
@@ -17,6 +17,10 @@ const compareValues = (a, b) => {
   return 0;
 };
 
+export const needConfirmation = story =>
+  story.state === status.ACCEPTED ||
```

Comment:
There is one more case to consider, which are release type stories that are gonna be released. Check the link from the other comment.